message-streams/javascript/package.json

```diff
@@ -2,17 +2,34 @@
   "name": "@cucumber/message-streams",
   "version": "3.0.0",
   "description": "Streams for reading/writing messages",
-  "main": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "type": "module",
+  "main": "dist/cjs/src/index.js",
+  "types": "dist/cjs/src/index.d.ts",
+  "files": [
+    "dist/cjs",
+    "dist/esm"
+  ],
+  "module": "dist/esm/src/index.js",
+  "jsnext:main": "dist/esm/src/index.js",
```

Comment:
What is `jsnext:main` for?

Reply:
That is some legacy, it was before the "module" entry-point.
I remove it. I guess this is useless now as "module" is supported since at least node 12.